crates/chia-tools/src/bin/run-spend.rs

```diff
@@ -135,7 +136,7 @@ fn print_puzzle_info(a: &Allocator, puzzle: NodePtr, solution: NodePtr) {
         return;
     };
 
-    match tree_hash(a, uncurried.program) {
```

Comment:
should `tree_hash()` return a `TreeHash`? it would seem to strengthen type-safety